internal/services/formatters/php/phpcs/formatter.go

```diff
@@ -94,20 +96,27 @@ func (f *Formatter) parseResults(results map[string]interface{}) {
 
 func (f *Formatter) parseMessages(filepath string, result interface{}) {
 	for _, message := range f.parseToResult(result).Messages {
-		if message.IsValidMessage() {
-			f.AddNewVulnerabilityIntoAnalysis(f.setVulnerabilityData(filepath, message))
+		if message.IsAllowedMessage() &&
+			!strings.Contains(message.Message, "This implies that some PHP code is not scanned by PHPCS") {
+			vuln := f.setVulnerabilityData(filepath, message)
+			f.AddNewVulnerabilityIntoAnalysis(vuln)
 		}
 	}
 }
 
 func (f *Formatter) setVulnerabilityData(filepath string, result entities.Message) *vulnerability.Vulnerability {
 	vuln := f.getDefaultVulnerabilitySeverity()
-	vuln.Severity = severities.Unknown
+	vuln.Severity = result.GetSeverity()
 	vuln.Details = result.Message
 	vuln.Line = result.GetLine()
 	vuln.Column = result.GetColumn()
 	vuln.File = f.RemoveSrcFolderFromPath(filepath)
 	vuln = vulnhash.Bind(vuln)
+
+	// Note: this code was added after of the bind for not generated breaking changes in vulnerability hash
```

Comment:
This comment is kind hard to understand.

Something like this will be better:

`NOTE: code and details were set after the vulnhash.Bind to avoid changes in the hash generation`

Reply:
done :)